plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -50,6 +57,21 @@ func getMaxVols(defaultVal int) int {
 	return defaultVal
 }
 
+// getHardPodAffinityWeight checks the hard pod affinity weight environment variable, otherwise returning a default value
```

Comment:
It's probably better to do this as a command-line flag rather than as an environment variable. Something like
--hard-pod-affinity-symmetric-weight